<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservation</class>
 <widget class="QMainWindow" name="reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="make_reservation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Make Reservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_reservation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Reservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_reservation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Reservation</string>
    </property>
   </widget>
   <widget class="QLabel" name="reservation_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
